src/main/java/creativei/entity/InquiryGuardian.java

```diff
@@ -16,6 +16,7 @@
     public InquiryGuardian(){}
 
     public InquiryGuardian(GuardianVo guardianVo){
+        this.setId(guardianVo.getId());
         this.name=guardianVo.getName()==null||guardianVo.getName().isEmpty()?null:guardianVo.getName();
```

Comment:
Use StringUtils.isEmpty

Reply:
sir can i make a method in request helper to check if vo field is empty or not?

Reply:
Instead, create a StringUtil class of our own and inside that add these helper methods.

Reply:
created in commit 44f49a9 .Please review.